samples/edgehog_app/prj.conf

```diff
@@ -40,12 +40,10 @@ CONFIG_MBEDTLS_ENABLE_HEAP=y
 CONFIG_MBEDTLS_HEAP_SIZE=95000
 # 16384 is the max absolute value, could be set much lower
 CONFIG_MBEDTLS_SSL_MAX_CONTENT_LEN=16384
-CONFIG_MBEDTLS_PEM_CERTIFICATE_FORMAT=y # Does not include writing
-CONFIG_MBEDTLS_PK_WRITE_C=y # Required for PEM writing (also needs MBEDTLS_PEM_WRITE_C)
-CONFIG_MBEDTLS_ENTROPY_ENABLED=y
-CONFIG_MBEDTLS_ZEPHYR_ENTROPY=y
-CONFIG_MBEDTLS_USER_CONFIG_ENABLE=y
-CONFIG_MBEDTLS_USER_CONFIG_FILE="../../astarte-device-sdk-zephyr/samples/astarte_app/include/tls-conf.h"
+CONFIG_MBEDTLS_PEM_CERTIFICATE_FORMAT=y
+CONFIG_MBEDTLS_PK_WRITE_C=y # Required for PEM writing
+CONFIG_MBEDTLS_ENTROPY_C=y
```

Comment:
These options seems generate some warnings to integration tests
```
warning: attempt to assign the value 'y' to the undefined symbol MBEDTLS_ENTROPY_ENABLED
warning: attempt to assign the value 'y' to the undefined symbol MBEDTLS_ZEPHYR_ENTROPY
```

Reply:
Ups, I forgot to change the settings in the tests project config file